app/src/main/AndroidManifest.xml

```diff
@@ -10,15 +10,49 @@
     android:roundIcon="@mipmap/ic_launcher_round"
     android:supportsRtl="true"
     android:theme="@style/OppiaTheme">
-    <activity android:name="org.oppia.app.HomeActivity">
+    <activity
+      android:name=".drawer.ui.preferences.PreferencesActivity"
+      android:label="@string/title_activity_navigation_drawer"
+      android:theme="@style/OppiaTheme.NoActionBar">
+<!--      <intent-filter>-->
+<!--        <action android:name="android.intent.action.MAIN" />-->
+
+<!--        <category android:name="android.intent.category.LAUNCHER" />-->
+<!--      </intent-filter>-->
+<!--      <intent-filter>-->
+<!--        <action android:name="android.intent.action.VIEW" />-->
+
+<!--        <category android:name="android.intent.category.DEFAULT" />-->
+<!--      </intent-filter>-->
+    </activity>
+    <activity
+      android:name=".ParentActivity"
+      android:label="@string/title_activity_navigation_drawer"
+      android:theme="@style/OppiaTheme.NoActionBar">
+<!--      <intent-filter>-->
+<!--        <action android:name="android.intent.action.MAIN" />-->
+
+<!--        <category android:name="android.intent.category.LAUNCHER" />-->
+<!--      </intent-filter>-->
+<!--      <intent-filter>-->
+<!--        <action android:name="android.intent.action.VIEW" />-->
+
+<!--        <category android:name="android.intent.category.DEFAULT" />-->
+<!--      </intent-filter>-->
+    </activity>
+    <activity android:name=".HomeActivity"
+      android:theme="@style/OppiaTheme.NoActionBar">
```

Comment:
What are these comments for? Do we need to retain them? If so, write inline comments accordingly, maybe try answering why this code block is commented